specification/ai/Azure.AI.Projects/evaluation-results/routes.tsp

```diff
@@ -56,5 +56,18 @@ interface EvaluationResults
     EvaluationResult,
     AssetCredentialRequest,
     AssetCredentialResponse
-  >; 
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Get agent evaluation results.")
+  @route("/runs/agents/{runId}")
+  @get
+  getAgentEvaluationResults is Azure.Core.Foundations.Operation<
```

Comment:
This is a bit confusing to me. If this is to get results, maybe we should have  "results" in the route? From the route, as a user, I expect it to return run details

Separately, is this supposed to converge with evaluation result asset post Build?

Reply:
the name getAgentEvaluationResults is still confusing based on your description then if this API is going to be exposed to users. Approved but this needs to be fixed before it goes public